nx-dev/ui-home/src/lib/smarter-tools-for-monorepos.tsx

```diff
@@ -0,0 +1,1079 @@
+  ButtonLink,
+  SectionHeading,
+  Strong,
+  TextLink,
+} from '@nx/nx-dev/ui-common';
+import { cx } from '@nx/nx-dev/ui-primitives';
+import { motion } from 'framer-motion';
+import {
+  AngularIcon,
+  AstroIcon,
+  CypressIcon,
+  DotNetIcon,
+  EslintIcon,
+  ExpoIcon,
+  ExpressIcon,
+  FastifyIcon,
+  GoIcon,
+  GradleIcon,
+  JavaIcon,
+  JestIcon,
+  JetBrainsIcon,
+  NestJSIcon,
+  NextJSIcon,
+  NodeIcon,
+  NuxtIcon,
+  PlaywrightIcon,
+  PnpmIcon,
+  QwikIcon,
+  ReactIcon,
+  RemixIcon,
+  RollupIcon,
+  RspackIcon,
+  RustIcon,
+  SolidIcon,
+  StorybookIcon,
+  TypeScriptIcon,
+  VisualStudioCodeIcon,
+  ViteIcon,
+  VueIcon,
+  WebpackIcon,
+} from '@nx/nx-dev/ui-icons';
+
+export function SmarterToolsForMonorepos(): JSX.Element {
+  return (
+    <section className="bg-slate-50 py-32 shadow-inner sm:py-40 dark:bg-slate-900">
+      <article className="mx-auto max-w-7xl px-6 lg:px-8">
+        <div className="max-w-5xl">
+          <SectionHeading
+            as="h2"
+            variant="title"
+            id="smarter-tools"
+            className="scroll-mt-24"
+          >
+            Smarter tools for your monorepo.
+          </SectionHeading>
+          <SectionHeading as="p" variant="subtitle" className="mt-6">
+            Nx Plugins are{' '}
+            <Strong>like VSCode extensions, but for your Nx workspace</Strong>.
+            Imagine your{' '}
+            <TextLink
+              href="/concepts/inferred-tasks?utm_source=homepage?utm_medium=website&utm_campaign=homepage_links&utm_content=cta_smarter_tools"
+              target="_blank"
+              rel="noopener"
+              title="Inferred tasks"
+            >
+              tools being automatically detected
+            </TextLink>
+            , with{' '}
+            <TextLink
+              href="/features/cache-task-results?utm_source=homepage?utm_medium=website&utm_campaign=homepage_links&utm_content=cta_smarter_tools"
+              title="Caching"
+              target="_blank"
+              rel="noopener"
+            >
+              caching
+            </TextLink>
+            ,{' '}
+            <TextLink
+              href="/ci/features/distribute-task-execution?utm_source=homepage?utm_medium=website&utm_campaign=homepage_links&utm_content=cta_smarter_tools"
+              title="Task distribution"
+              target="_blank"
+              rel="noopener"
+            >
+              task distribution
+            </TextLink>
+            , and{' '}
+            <TextLink
+              href="/ci/features/distribute-task-execution?utm_source=homepage?utm_medium=website&utm_campaign=homepage_links&utm_content=cta_smarter_tools"
+              title="Task distribution"
+              target="_blank"
+              rel="noopener"
+            >
+              task splitting
+            </TextLink>{' '}
+            being pre-configured for you. All in sync with your underlying
+            tooling configuration. Plus there's{' '}
```

Comment:
`Plus there are` (grammar, and consistency with the lack of contraction elsewhere on the page)